src/indev/lv_indev_gesture.h

```diff
@@ -0,0 +1,115 @@
+/**
+ * @file lv_indev_gesture.h
+ * @author EDGEMTech Ltd, erik.tagirov@edgemtech.ch
+ */
+
+#ifndef LV_INDEV_GESTURE_H
+#define LV_INDEV_GESTURE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../core/lv_obj.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+#define LV_GESTURE_MAX_TOUCHES 4 
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/* The tates of a gesture recognizer */
+typedef enum {
+	LV_INDEV_GESTURE_STATE_NONE = 0, 	/* Beginning & end */
+	LV_INDEV_GESTURE_STATE_ONGOING,		/* Set when there is a probability */
+	LV_INDEV_GESTURE_STATE_RECOGNIZED,	/* Recognized, the event will contain touch info */
+	LV_INDEV_GESTURE_STATE_CANCELED,	/* Canceled - usually a finger is lifted */
+} lv_indev_gesture_state_t;
+
+/* Data structures for touch events - used to repsensent a libinput event */
+/* Emitted by devices capable of tracking identifiable contacts (type B) */
+typedef struct {
+	lv_point_t point;		/* Coordinates of the touch */
+	lv_indev_state_t state;		/* The state i.e PRESSED or RELEASED */
+	uint8_t id;			/* Identification/slot of the contact point */
+	uint32_t ts;			/* Timestamp in milliseconds */
```

Comment:
Avoid unclear abbreviations
```suggestion
	uint32_t timestamp;			/* Timestamp in milliseconds */
```